### ISSUE TITLE
Footer examples appear to be showing invalid HTML

### ISSUE THREAD

**Issue by tbaxter-18f** [uswds/uswds#2379]
In the footer examples (https://designsystem.digital.gov/components/footers/) we are showing the use of the contentinfo role:
<footer class="usa-footer usa-footer-big" role="contentinfo">

I believe this is incorrect and should be removed, as the footer in that scenario implicitly has contentinfo semantics. See https://www.w3.org/TR/html-aria/

It doesn't impact functionality (although it may be slightly wobbly a11y-wise), but I do think we have an obligation to provide the best examples we can.

References:
HTML5Doctor (2015) http://html5doctor.com/on-html-belts-and-aria-braces/
Sitepoint (2016) https://www.sitepoint.com/how-to-use-aria-effectively-with-html5/
Paciello group (2016): https://developer.paciellogroup.com/blog/2016/10/a-not-so-short-note-on-aria-to-the-rescue/
W3C (Last updated 2018): https://www.w3.org/TR/html-aria/#docconformance

**Comment by maya** [comment]
We want to ensure we support the people using various AT. According to Deque University, it's the best practice to use both: https://dequeuniversity.com/assets/html/jquery-summit/html5/slides/landmarks.html#best_practice_use_both

That is also echoed here: https://html5forwebdesigners.com/today/#aria

---
### I had a conversation on the web accessibility Slack to get clarity.
However, there's not a strong consensus.

kivi [10:01 AM]
It's hard to argue with the W3C.

ghanek [10:04 AM]
Ideally we wouldn't have a need to be redundant. Unfortunately, we can't ignore the current states of IE and Edge, either, and their development roadmaps & timelines. (edited)

mayabenari [10:06 AM]
@ghanek Thanks, I'm not familiar with their roadmaps, can you clarify what that implies wrt whether we need to be redundant or not?
Also, are there any issues with landmark support on older screen readers that would be improved by being redundant?

erikwesslen [10:43 AM]
joined #general.

Just Scott [10:48 AM]
@mayabenari - just released an article about landmarks last weekend http://www.scottohara.me/blog/2018/03/03/landmarks.html (edited)

karlgroves [10:48 AM]
> According to Deque University, it’s the best practice to use both

That’s out of date IMO

mayabenari [10:50 AM]
@Just Scott that's really helpful, thanks... Is the tl;dr don't use both except for `<footer role="contentinfo"></footer>` which has a known bug?

Just Scott [10:54 AM]
basically yes.  Test your landmarks in the browsers / AT you support.  if you find something doesn't gel, double up if need be.  Otherwise you shouldn't need to duplicate inherent role with ARIA role

dylanb [10:57 AM]
@kivi very easy to argue with the W3C, just because you write a spec, it does not mean you can use what that spec describes. It takes action by a lot of people first.

dylanb [11:00 AM]
@mayabenari the reason that we recommend that is to allow for support in older ATs. What is the negative impact of this in browsers that support ARIA 1.1 ?

Nothing!

What is the potential benefit to users with disabilities who do not have the money to upgrade their expensive AT?

Potentially huge! (edited)
That is the definition of a “best practice” IMO
